cg/apps/lims/api.py

```diff
@@ -388,3 +388,32 @@ def get_sample_comment(self, sample_id: str) -> str:
     def get_sample_project(self, sample_id: str) -> str:
         """Get the lims-id for the project of the sample"""
         return self.sample(sample_id).get("project").get("id")
+
+    def get_sample_rin(self, sample_id: str) -> float:
+        """Get the sample RIN value determined in the Initial QC steps"""
```

Comment:
```suggestion
        """Return the sample RIN value."""
```
This might change in LIMS and then this will be confusing. Too much knowledge of another system